code/weapon/weapon.h

```diff
@@ -385,7 +434,7 @@ struct weapon_info
 
 	float	damage;								//	damage of weapon (for missile, damage within inner radius)
 	float	damage_time;						// point in the lifetime of the weapon at which damage starts to attenuate. This applies to non-beam primaries. (DahBlount)
-	int   damage_curve_idx;					// damage over time curve
+	int		damage_curve_idx;
```

Comment:
Any reason you couldn't update this one like the others?

Reply:
For damage, I would want things like hit angle and target health as inputs, so that needs to go in a separate batch. (I went back and forth on this a lot.)